portfolio/src/main/webapp/index.html

```diff
@@ -61,4 +53,24 @@ <h1>Naomi Amuzie's Portfolio</h1>
       <p>Click <a href="https://github.com/ncaa2018">here</a> for my GitHub profile. </p>
     </div>
   </body>
+
+
+    <body>
+    <div id="content">
```

Comment:
```suggestion
```

I think it makes more sense to have one <body><div id="content"></div></body> set.